libs/status/README.md

```diff
@@ -0,0 +1,24 @@
+# HASH Status and Error Model
+
+## Goals
+
+A consistent structure and pattern for producing API-agnostic definitions of statuses and errors.
+
+- Services should be able to understand and handle error conditions of other service’s responses
```

Comment:
nit: should this be the following?
```suggestion
- Services should be able to understand and handle error conditions of other services' responses
```

Reply:
It should be, thank you

Reply:
[Grammar](https://github.com/hashintel/hash/pull/1787/commits/31215c0ab7d069e6728eba81528eb9eee2f7c0ea)